adwords_api/lib/adwords_api/report_utils.rb

```diff
@@ -218,6 +219,17 @@ def check_for_xml_error(report_body, response_code)
       end
     end
 
+    def check_for_rate_exceeded_error(report_body, response_code)
```

Comment:
Add a brief comment describing the method so we stay consistent with the rest of the file.